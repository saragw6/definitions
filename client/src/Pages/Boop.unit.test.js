import React from 'react';
import renderer from 'react-test-renderer';
import Boop from './Boop';

test('works?', () => {
  const component = renderer.create(<Boop/>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});