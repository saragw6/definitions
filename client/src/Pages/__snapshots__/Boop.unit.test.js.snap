exports[`test works? 1`] = `
<div>
  hello
</div>
`;
